nextflow.config

```diff
@@ -1,7 +1,7 @@
 // these values supersede params in the profile and are themselves
 // superseded by contents of -params-file
 params {
-    container = 'quay.io/nhoffman/dada2-nf:v1.12-9-g39fa45b'
+    container = 'dada2-nf:v1.12-153-g4a48721-dirty'
```

Comment:
@nhoffman -  We'll need to update your image on quay.  See docker/install_infernal_and_easel.sh (above)